http/responses.go

```diff
@@ -56,3 +55,54 @@ func doWriteXml(w http.ResponseWriter, code int, v any) error {
 
 	return nil
 }
+
+// OkHTML writes v into w with 200 OK.
+func OkHTML(w http.ResponseWriter, v any) {
+	WriteHTML(w, http.StatusOK, v)
+}
+
+// OkHTMLCtx writes v into w with 200 OK.
+func OkHTMLCtx(ctx context.Context, w http.ResponseWriter, v any) {
+	WriteHTMLCtx(ctx, w, http.StatusOK, v)
+}
+
+// WriteHTML writes v as HTML string into w with code.
+func WriteHTML(w http.ResponseWriter, code int, v any) {
+	if err := doWriteHTML(w, code, v); err != nil {
+		logx.Error(err)
+	}
+}
+
+// WriteHTMLCtx writes v as HTML string into w with code.
+func WriteHTMLCtx(ctx context.Context, w http.ResponseWriter, code int, v any) {
+	if err := doWriteHTML(w, code, v); err != nil {
+		logx.WithContext(ctx).Error(err)
+	}
+}
+
+func doWriteHTML(w http.ResponseWriter, code int, v any) error {
```

Comment:
Why not show that the definition type is string?

Reply:
good idea

Reply:
@kesonan FIX

Reply:
@kesonan ken fix